src/universal/components/CodeEditor/CodeEditor.js

```diff
@@ -4,12 +4,14 @@ export default class CodeEditor extends Component {
   static propTypes = {
     mode: PropTypes.any,
     value: PropTypes.string,
+    readOnly: PropTypes.bool,
```

Comment:
the name of the property is the same like in the CodeMirror object. I kept the same name for consistency.